app/services/force/soql_query_builder.rb

```diff
@@ -0,0 +1,122 @@
+module Force
+  class SoqlQueryBuilder
```

Comment:
@callaghanc @creativecoder This is the code that I wanted you to review. This is SQOL builder. I thought about doing it this way so we can A) hide SOQL details b) have flexibility on building queries without focusing in SOQL C) incorporate other functionally like pagination (since now I have the from clause from one side and the select in the other and I can run the count). This builder would the basic block on top of `Restforce`.

Reply:
I like this approach and the API you have seems intuitive to me. Looks good!

Reply:
👍